WSDebugger.pas

```diff
@@ -65,15 +65,21 @@ procedure OutputDebugString(const AMsg: string);
 {$IF DEFINED(DEBUG) OR DEFINED(DEBUG_WS) OR DEFINED(CHECKSPEED)}
   LDateTime := FormatDateTime('hh:nn:ss', Now);
   LMsg := Format('%s - %s', [LDateTime, AMsg]);
-  {$IF DEFINED(MSWINDOWS) OR DEFINED(DEBUG_WS)}
-  Winapi.Windows.OutputDebugString(PChar(LMsg));
-  {$ENDIF}
-  {$IF DEFINED(ANDROID)}
-   if TPlatformServices.Current.SupportsPlatformService(IFMXLoggingService, LLogService) then
-     begin
-       LLogService.Log('%s', [LMsg]);
-     end;
-  {$ENDIF}
+  {$IF DEFINED(DEBUG_WS)}
+    {$IF DEFINED(MSWINDOWS)}
+    Winapi.Windows.OutputDebugString(PChar(LMsg));
+    {$ELSE}
+      {$IF DEFINED(ANDROID)}
+       if TPlatformServices.Current.SupportsPlatformService(IFMXLoggingService, LLogService) then
+         begin
+           LLogService.Log('%s', [LMsg]);
+         end;
+      {$ENDIF}
+      {$IF DEFINED(LINUX)}
+        Writeln(lMsg);
+      {$ENDIF}
+    {$ENDIF}
```

Comment:
Instead of 
{$IF DEFINED(MSWINDOWS)}
{$ELSE}
   {$IF DEFINED(ANDROID)}
   {$ENDIF}
    {$IF DEFINED(LINUX)}
    {$ENDIF}
{$ENDIF}
  
use

{$IF DEFINED(MSWINDOWS)}
{$ELSEIF DEFINED(ANDROID)}
{$ELSEIF DEFINED(LINUX)}
{$ENDIF}